include/wolfboot/wolfboot.h

```diff
@@ -195,7 +195,11 @@ extern "C" {
 
  /* KeyStore API */
  int keystore_num_pubkeys(void);
+#if defined(WOLFBOOT_RENESAS_SCEPROTECT)
```

Comment:
don't do this, just cast your buffer to uint32_t* in one place if you need it in that form